apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/FhirAppConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package gov.cms.bfd.server.war;
+
+import java.io.Serializable;
+import java.util.Optional;
+
+/** Models the configuration options for the application. */
+public final class FhirAppConfiguration implements Serializable {
```

Comment:
From previous PR, notes from @karlmdavis 

> The application's configuration is all currently handled by the SpringConfiguration class. If you look, you can note that several of the properties in there are injected via environment variables.
> 
> I think you should probably remove this class, and just use something new in SpringConfiguration, instead.

Feels like a great suggestion, so we don't duplicate how / where we are supplying app configuration. I was previously unaware of this piece of the code. Here's a link to the relevant code: https://github.com/CMSgov/beneficiary-fhir-data/blob/af23b397cac9c7d265237c123bd473ddcf35d07e/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java#L50

Reply:
DONE